rqt_gui_cpp/CMakeLists.txt

```
@@ -23,22 +43,27 @@ set(rqt_gui_cpp_HDRS
 
 qt5_wrap_cpp(rqt_gui_cpp_MOCS ${rqt_gui_cpp_HDRS})
 
-add_library(${PROJECT_NAME} ${rqt_gui_cpp_SRCS} ${rqt_gui_cpp_MOCS})
-target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES} Qt5::Widgets)
+ament_export_dependencies(Qt5Widgets)
 
-if(APPLE)
-  set_target_properties(${PROJECT_NAME} PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
-endif()
+ament_export_include_directories(include)
 
-install(FILES plugin.xml
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
-)
+add_library(${PROJECT_NAME}
+  ${rqt_gui_cpp_SRCS}
+  ${rqt_gui_cpp_MOCS})
 
-install(TARGETS ${PROJECT_NAME}
-  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-)
+target_link_libraries(
+  ${PROJECT_NAME}
+  ${qt_gui_cpp_LIBRARIES}
+  ${ament_LIBRARIES}
+  Qt5::Widgets
+  ${TinyXML_LIBRARIES})
 
-install(DIRECTORY include/${PROJECT_NAME}/
-  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+install(
+  DIRECTORY include/
+  DESTINATION include
 )
+
+install(FILES plugin.xml
+  DESTINATION share)
```

Package specific destination
```suggestion
  DESTINATION share/${PACKAGE_NAME})
```